src/go/plugin/go.d/agent/discovery/sd/discoverer/snmpsd/sysinfo.go

```diff
@@ -39,16 +39,15 @@ func GetSysInfo(client gosnmp.Handler) (*SysInfo, error) {
 		Organization: "Unknown",
 	}
 
-	r := strings.NewReplacer("\n", "\\n", "\r", "\\r")
+	r := strings.NewReplacer("\n", " ", "\r", " ")
```

Comment:
You replaced `\n` with a space one time too often.

Reply:
Oops never mind

Reply:
What?